src/masonite/providers/SessionProvider.py

```diff
@@ -13,10 +13,9 @@ def register(self):
         session = Session(self.application).set_configuration(config("session.drivers"))
         session.add_driver("cookie", CookieDriver(self.application))
         self.application.bind("session", session)
-        self.application.make("view").share({"old": self.old})
 
     def boot(self):
-        pass
+        self.application.make("view").share({"old": self.old})
```

Comment:
why is this moved into the boot?

Reply:
I don't know lol I was thinking that it was the correct place to define view sharing...

Reply:
do you want to revert this ?

Reply:
I don't think it is right? Well it depends. In this case the old method is not called until you call it in your templates. 

In this case it will make each request slower because it has to register this old method every request. 

If we keep it in the register method it only gets registered once and it should still work